### DISCUSSION TITLE
Surface Hub Support

### DISCUSSION THREAD

**Discussion by gibbyhome** [rocksdanister/lively#916] [Ideas]
Love this app, use it on all my machines but it would look awesome on a surface hub screen.. is  there any work on Hub support

**Comment by rocksdanister** [comment]
Hardware seems capable, looks like the issue is the custom Windows 10 Team operating system?.. which is a rebrand of Windows 10 Enterprise.

Did you get an error message when trying to run the installer? 
Did you try the Windows store version of Lively?

**Reply by gibbyhome** [reply]
since it is windows 10 team the app does not show up in the microsoft store unless its hub supported. I'm not sure if I can install it as an executable but I can try.

**Reply by rocksdanister** [reply]
The installer version of Lively require Windows 10 version 1903 or above .. I don't know if Enterprise (teams) is still on 1809.

I can try enabling surface hub support in store if you can help me test it.

**Reply by gibbyhome** [reply]
I would love to help test it, our hubs are on the latest versions, in in the US so in can test between 9am - 5pm

**Reply by rocksdanister** [reply]
Ok, send me an email at awoo.git@gmail.com and we can figure something out
... probably an internal build of Lively for testing.
